assets/script.js

```diff
@@ -24,4 +24,27 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
     alertMsg.innerText = '* Please Enter Your Email Only In Lower Case';
   }
-});
\ No newline at end of file
+});
+
+// data preservation inside the browser
+
+const dataTotal = {};
+document.addEventListener('keyup', () => {
+  dataTotal.name = document.querySelector('.fname-input-fld').value;
+  dataTotal.email = document.querySelector('.email-input-fld').value;
+  dataTotal.text = document.querySelector('.text-input-fld').value;
+  const allData = JSON.stringify(dataTotal);
+  localStorage.setItem('dataTotal', allData);
+});
+
+const storeItem = () => {
+  const parsedData = JSON.parse(localStorage.getItem('dataTotal'));
+  const fname = document.querySelector('.fname-input-fld');
+  const email = document.querySelector('.email-input-fld');
+  const text = document.querySelector('.text-input-fld');
+  fname.value = parsedData.name;
+  email.value = parsedData.email;
+  text.value = parsedData.text;
+};
+
+window.onload = storeItem;
```

Comment:
I can see you have put in a lot of effort in setting up your local storage to hold data👍 however there are still some issues😉

- [ ] The contact form information is not persistent when the page is reloaded i.e. the contact form is not being filled automatically with the data from the local storage on page reload😉
- [ ] The local storage is still not being populated with the data from the form😉

These are necessary [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md) for this milestone, kindly check your code again and fix the issue🤗
<img width="799" alt="local_data" src="https://user-images.githubusercontent.com/64643995/218275899-dd730f5e-7d37-48ab-bb5e-e27a1d11409a.png">